tests/dummy/config/environment.js

```diff
@@ -41,11 +40,11 @@ module.exports = function(environment) {
     ENV.APP.LOG_VIEW_LOOKUPS = false;
 
     ENV.APP.rootElement = '#ember-testing';
+    ENV.APP.autoboot = false;
   }
 
-  if (environment === 'production') {
+  if (environment === 'production') {} // eslint-disable-line
 
```

Comment:
We should put a comment in this block saying it is intentionally an empty block, rather than disabling lint.